server/metrics/metrics.go

```diff
@@ -14,6 +14,9 @@ const (
 	// CRIOOperationsLatencyKey is the key for the operation latency metrics.
 	CRIOOperationsLatencyKey = "crio_operations_latency_microseconds"
 
+	// CRIOOperationsLatencyPerCRICallKey is the key for the operation latency metrics for each CRI call.
+	CRIOOperationsLatencyPerCRICallKey = "crio_operations_latency_per_cri_call_microseconds"
```

Comment:
I think this should really be named `crio_operations_latency_microseconds` and the older one modified.

Reply:
I was thinking the same but was trying to avoid changing the old one. Any name suggestions for the old one? I was thinking `crio_operations_latency_microseconds_total`.

Reply:
Fixed